app/components/article-map.js

```diff
@@ -0,0 +1,26 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+
+  classNames: ['article-map'],
+
+  buildMap: function () {
+    L.mapbox.accessToken = 'pk.eyJ1IjoiYm1yc255IiwiYSI6IjlvWUIxYmcifQ.8iPZd41vr5xtnuenlkWwtw';
+    // Replace 'examples.map-i87786ca' with your map id.
+    let mapboxTiles = L.tileLayer('https://{s}.tiles.mapbox.com/v4/examples.map-i87786ca/{z}/{x}/{y}.png?access_token=' + L.mapbox.accessToken, {
```

Comment:
Line is too long.<br>'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'L' is not defined.